apps/ui/src/env.mjs

```diff
@@ -19,6 +19,9 @@ export const env = createEnv({
     NEXT_OUTPUT: z.string().optional(),
     NEXT_IMAGES_UNOPTIMIZED: optionalZodBoolean,
     NODE_ENV: z.string().optional(),
+    SENTRY_AUTH_TOKEN: z.string().optional(),
+    SENTRY_ORG: z.string().optional(),
+    SENTRY_PROJECT: z.string().optional(),
```

Comment:
Wouldn't it be better to make this an union? If I understand it correctly, if you have one, others should be setup as well.

Reply:
Yes, you are correct. We would have to use more complex validation here (eg `refine()`).. for now I prefer keep it simple